pkgdb/Makefile

```diff
@@ -65,10 +63,8 @@ REPO_ROOT    := $(patsubst %/,%,$(dir $(MAKEFILE_DIR)))
 # Install Prefixes
 # ----------------
 
-PREFIX     ?= $(PKGDB_ROOT)/build
-BINDIR     ?= $(PREFIX)/bin
-LIBEXECDIR ?= $(PREFIX)/libexec
-SYSCONFDIR ?= $(PREFIX)/etc
+PREFIX ?= $(PKGDB_ROOT)/build
+BINDIR ?= $(PREFIX)/bin
```

Comment:
It still confuses me that judging from these lines we build our binaries into a `build/bin` directory, but actually it still ends up in `/bin` or `/out/bin` if we run `make install`.

Reply:
`PREFIX` is set by `nix` and represents the _install prefix_.

That directory will not appear locally anymore though after this merges.